source/includes/_configuration-options.md

```diff
@@ -208,3 +263,20 @@ If there is a need to use custom css stylesheet inside the Tipser dialog iframe,
 1. set `useCustomCss` parameter to **true**
     
 2. Send the custom css stylesheet to Tipser administrator in order for it to be uploaded to your account.
+
+### updateConfig()
+
+To update any of the configuration options dynamically, please use the `updateConfig()` function. 
+
+* In Tipser Script:
+
+```javascript
+window.elements = tipser.updateConfig({
+  prePopulatedAddress: {
+    delivery: {
+      email: 'youremail@email.com',
```

Comment:
Czy ten kod w ogóle zadziała?

Reply:
Aktualnie opcjonalne wartości są na devie, może rzeczywiście lepiej na razie dodać w docsach puste stringi i zmieni to jak już BE wrzuci tą zmianę na prod.